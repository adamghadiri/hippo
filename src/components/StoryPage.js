import React from "react";
import Story from "./Story";

export default (props) => (
  <Story showComments={true} itemId={props.match.params.id} />
);
