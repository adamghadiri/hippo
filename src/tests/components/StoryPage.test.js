import React from "react";
import { shallow } from "enzyme";
import StoryPage from "../../components/StoryPage";

test("should render story page correctly", () => {
  const match = { params: { id: "testItemId" } };
  const wrapper = shallow(<StoryPage match={match} />);
  expect(wrapper).toMatchSnapshot();
});
