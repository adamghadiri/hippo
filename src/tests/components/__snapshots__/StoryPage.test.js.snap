// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render story page correctly 1`] = `
<ConnectFunction
  itemId="testItemId"
  showComments={true}
/>
`;
